### ISSUE TITLE
cannot conect to wifi

### ISSUE THREAD

**Issue by ARKopp** [guysoft/OctoPi#309]
I cannot get my wifi working and connect with ssh. 
the octopi-network.txt does not seem to work. Only when I type in everything in like this below I get an ip adress but then i cannot connect with ssh. 

sudo nano /etc/network/interfaces

# WLAN
allow-hotplug wlan0
iface wlan0 inet manual
wpa-ssid "WLAN-NAME"
wpa-psk "WLAN-PASSWORT"

**Comment by guysoft** [comment]
What version of OctoPi are you using?

**Comment by WinDroidGuy** [comment]
I'm using the latest on the site 0.13 with the same issue, if I connect via ethernet it works fine; but my modem/router are in a completely different part of the house for that to work.

**Comment by shanehull** [comment]
Same issue here.

I'm using the latest image and have confirmed that it's not a text editor issue.

**Comment by WinDroidGuy** [comment]
@shanehull which OS are you using to edit?

**Comment by shanehull** [comment]
I'm using nano SSH'd into the Pi @WinDroidGuy.

**Comment by shanehull** [comment]
Tried a fresh image and it just worked!

**Comment by breadfan13** [comment]
Try this, then do the edit to your network interfaces you referenced the problem could be with how the " quotes are saving. or try using the @ sign if you don't want to change the locale

allow-hotplug wlan0
iface wlan0 inet manual
wpa-ssid @WLAN-NAME@
wpa-psk @WLAN-PASSWORT@

http://rohankapoor.com/2012/04/americanizing-the-raspberry-pi/

**Comment by JCacho** [comment]
Has anyone discovered another solution? Still can't connect it.

**Comment by shanehull** [comment]
Nope. Mine ended up connecting but never gets an IP. When you run iwconfig does the adapter say it's connected to your assigned SSID?

**Comment by John-Mc** [comment]
@JCacho   -  If the iwconfig command does not show you are connected, the next thing to try is:
`sudo iwlist wlan0 scan`  
Buried in the resulting output, you should see the SSIDs of any WiFi networks that your Pi finds (even if it is not connected to them). This will tell us if your Pi can even see your network.

If it sees your network but is not connecting to it, one of the more common problems (especially if you are using Apple's TextEdit on a Mac to edit the connection information) is not having the correct type of quotation marks around your SSID and password. Take a close look at your octopi-network.txt file. If the quotation marks around your SSID and password are curly, that is a problem. If you see that, the quick fix is to copy and paste the straight type of quotation marks from another part of the document. (We can also tell you how to shut off TextEdit's quotation mark substitution behavior, if this was the problem.)

**Comment by jormono** [comment]
Hello, new to github (and a complete novice when it comes to these types of things)here, I'm having a problem connecting to wifi as well. I've replaced the quotation marks from within the pi, and then rebooted. For some reason the scale of the screen seems to have changed and I couldn't see the results of the network scan, though I had done that before and saw my neighbors wifi network and wireless printer, but didn't see my own network (assumed it was off screen and couldn't figure out how to scroll and see it). I've tried every trick I know, at this point I'm wondering if it would be easier to start a fresh build from raspbian pixel and installing octoprint onto it. Any help is much appreciated!
